pkgs/development/python-modules/gudhi/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, cmake
+, boost
+, eigen
+, gmp
+, cgal_5  # see https://github.com/NixOS/nixpkgs/pull/94875 about cgal
+, mpfr
+, tbb
+, numpy
+, cython
+, pybind11
+, matplotlib
+, scipy
+, pytest
+, enableTBB ? false
+}:
+
+buildPythonPackage rec {
+  pname = "gudhi";
+  version = "3.4.1";
+
+  src = fetchFromGitHub {
+    owner = "GUDHI";
+    repo = "gudhi-devel";
+    rev = "tags/gudhi-release-${version}";
```

Comment:
```suggestion
    rev = "gudhi-release-${version}";
```

Reply:
That's the actual tag name in that project: not `gudhi-release-3.4.1` but `tags/gudhi-release-3.4.1`.

Reply:
ohh..